packages/vhd-lib/src/Vhd/VhdFile.js

```diff
@@ -416,7 +416,7 @@ export class VhdFile extends VhdAbstract {
   async _readParentLocatorData(parentLocatorId) {
     const { platformDataOffset, platformDataLength } = this.header.parentLocatorEntry[parentLocatorId]
     if (platformDataLength > 0) {
-      return (await this._read(platformDataOffset, platformDataLength)).buffer
+      return await this._read(platformDataOffset, platformDataLength)
```

Comment:
Can you explain this change and how it's related to this PR?

Reply:
that was one of the root cause of strange messages working on parent locator, notably on the .copy part to align data . The fun part is that Buffer.buffer exists https://nodejs.org/api/buffer.html#bufbuffer , it is the  underlying ArrayBuffer object based on which this Buffer object is created and most of the method accept it.
It's not completely in sync  with the buffer data, especially since we allocate small buffer ( less than poolSize 8182 bits)

Do you think it should be a separated commit or PR ?

Reply:
I think a separate commit would be good enough?